lib/repo/git_repository.rb

```diff
@@ -110,25 +110,10 @@ def self.create(connect_string)
       cloned_repo = Git.clone(
         'git@localhost:' + repo_name, MarkusConfigurator.markus_config_repository_storage + '/' + repo_name)
 
-      repo = Rugged::Repository.discover(MarkusConfigurator.markus_config_repository_storage + '/' + repo_name)
-
       # Lets make some sample files and the new master branch
       cloned_repo.reset
       cloned_repo.branch('master')
 
-      # Create the README.md on the filesystem
-      file_path_for_readme = File.join(repo.workdir, 'README.md')
-      File.open(file_path_for_readme, 'w+') do |readme|
-        readme.write('Initial commit.')
-      end
-
-      cloned_repo.add(all: true)
```

Comment:
It looks like the clone captures the file commits for the seeding.  So there is nothing to add and commit.  (Doing some more testing now.)